packages/gparted.rb

```diff
@@ -45,12 +47,24 @@ def self.build
   end
 
   def self.install
-    system "make DESTDIR=#{CREW_DEST_DIR} install"
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
   end
   
   def self.postinstall
-    puts "To run gparted, execute the following:".orange
-    puts "xhost si:localuser:root".lightblue
-    puts "sudo gparted".lightblue
+    bashrc = "#{HOME}/.bashrc"
+    puts 'Performing script-injection...'
+    unless `grep -c "# Added by gparted package" #{bashrc} || true`.to_i > 0
+      system "cat <<'EOF'>> #{bashrc}
+# Added by gparted package
+alias sudo='sudo '
+alias gparted='xhost si:localuser:root; sudo gparted'
```

Comment:
Why not create a bash script and include it in the package so it can run in any environment?

Reply:
Moved into script